cli/cmd/create_service.go

```diff
@@ -40,7 +40,7 @@ var crServiceCmd = &cobra.Command{
 			return errors.New("required argument SERVICENAME not set")
 		}
 
-		if !keptncommon.ValididateUnixDirectoryName(args[0]) {
```

Comment:
This comes from https://github.com/keptn/go-utils/blob/0c7d901bcee968fbdc9f471fb66d7fb265571432/pkg/lib/keptn/keptn_base.go#L222 so it can't be merged as is.

It's possible this is a "public" function that can never be fixed. But perhaps it can be.

I will drop this from this PR as it's too complicated for a generic spelling fix.

Reply:
I've moved this commit to here: https://github.com/jsoref/keptn/commit/spelling-validate

Reply:
This was fixed in #3324